mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -281,7 +282,7 @@ private Event dequeueEvent() {
         private final Duration TEN_SECONDS = Duration.ofSeconds(10);
 
         private static final int THREAD_COUNT = 10;
-        private static final int OPERATION_COUNT = 100;
+        private static final int OPERATION_COUNT = 100000;
```

Comment:
needed?

Reply:
yes, because if you can reach the MAX_EVENT_DISPATCHED_COUNT you don't test that you receiving events after the stop: they are all already delivered